backend/src/repository/dalBoard.ts

```diff
@@ -40,6 +40,15 @@ export const getByProject = async (projectID: string) => {
   }
 };
 
+export const getPersonal = async (projectID: string, ownerID: string) => {
+  try {
+    const board = await Board.findOne({ ownerID, projectID });
```

Comment:
I think it would also be worth adding a filter for the board scope (i.e. `Board.findOne({..., scope: BoardScope.PERSONAL})`) just so we 100% know we're getting a personal board back.